Linear_cell_complex/demo/Linear_cell_complex/CMakeLists.txt

```diff
@@ -16,6 +16,7 @@ endif()
 
 ## For profilling with gprof
 # add_definitions("-pg")
+add_definitions("-Wall -Wextra")
```

Comment:
That cannot be accepted into CGAL: you cannot decide which compiler flags the users may want. What is more, unless I read wrongly, you enable flags on all platforms, and that will creates compilation errors on all platforms that do not understand those flags (old versions of gcc, and all Microsoft compilers).